src/styles/app.css

```diff
@@ -63,5 +77,40 @@ a.qpp-docs-title-back {
 
 .qpp-docs-title-text {
   color: #FFFFFF;
-  font-weight: 600;
+}
+
+/* Design System Overrides */
+.ds-base {
+  font-family: Rubik, Arial, "Helvetica Neue", Helvetica, sans-serif;
+}
+
+.ds-h1, .ds-h2, .ds-h3, .ds-h4 {
+  font-weight: normal;
+}
+
+.ds-u-font-weight--semibold {
+  font-weight: 500!important;
+}
+
+a.ds-c-button:visited {
+  color: white;
+}
+
+/* New styles*/
+@media (max-width: 544px) {
+  #dropdown-nav {
+    display: block;
+  }
+  #left-nav {
+    display: none;
+  }
+}
+
+@media (min-width: 544px) {
```

Comment:
As an alternative to having these style declarations that toggles an element's visibility at different breakpoints, you can accomplish the same functionality with the ["display" utility class](https://design.cms.gov/utilities/display-visibility#utilities.display-visibility.display): 

The following would hide the element on viewports larger than 544px, and show it when the viewport is smaller than 544px.

```jsx
<div className='ds-u-display--block ds-u-sm-display--none'>
```